doc/user_guide/model.rst

```diff
@@ -574,22 +574,37 @@ spectrum at a particular point in a spectrum-image) use
 :py:meth:`~.model.BaseModel.fit`.
 
 The following table summarizes the features of the currently available
-optimizers:
-
```

Comment:
It'll be good to add `versionadded` and `versionchanged` sphinx's markup describing the changes.